### ISSUE TITLE
Genesis Solo Validator Not on the List

### ISSUE THREAD

**Issue by Tachyonica** [rated-network/solo-stakers#3]
Hi. I have validated from home since genesis with deposit address: 0xc4374C15b5d3e0f43125CC944Abf5D1411a4a0db. I cannot see why this solo staking node would be excluded from your list. Thanks for considering this issue!

**Comment by ariskk** [comment]
Hey! The approach is probabilistic, so we are bound to have false negatives. For more info: https://blog.rated.network/blog/solo-stakers

We are working on a v2, and any feedback we receive now will be used to fine tune the model.

**Comment by rambowtech** [comment]
Also missing:

**Comment by Tachyonica** [comment]
Can report a an independent operator, infrastructure on premises, self-funded, fewer than 200 validators, unique graffiti, validators in first 500k, etc. Would be interested to see why the 5-fold cross-validation technique ended up giving it such a low score. Perhaps one distinguishing feature of my node is "geographic diversity" - located in the far global South it experiences slightly higher latencies than average with a slight impact on performance. It would seem pretty unfortunate for the model to exclude nodes on that basis at least! :)

**Comment by Tachyonica** [comment]
Silver lining: if you missed me, maybe the estimate of 6.5% solo stakers is an under-estimate...

**Comment by nelway42** [comment]
Yep my validator is also missing.  Single deposit to a single validator, operating from home, index within the first 50k, funded from my own wallet. If I'm not a solo stalker I don't know who is!

**Comment by ProSoloStaker** [comment]
Hi, same here - I can’t figure out why validator #409907 is not in the list. Solo staker here with 2 validators, doing it directly without any third-party like Rocket Pool, Allnodes, etc.
Thanks!

**Comment by SenatorGrimm** [comment]
Validator 355870 is missing too. Activated April 2022, running as a single validator from home on Intel NUC with Ubuntu, Geth, Lighthouse.

**Comment by 0xkindatired** [comment]
Adding a comment here as I'm also missing from the list. I am but a humble solo staker running one validator at home (also Geth + Lighthouse). Hopefully mine shows up on the next version <3

Index #412309 with withdrawal address: 0xB4B65FCb010ce087E4B90eA73D9b313C93fC352D

**Comment by immanchand** [comment]
Adding mine here aswell. Index 12972 and 404848. Completely home solo staking and fulfills are the criteria mentioned in your blog.

**Comment by edrews** [comment]
Adding my missing solo staker address here using Prysm and Erigon: 0xE2EFFFfE7fB7015862cBb1958A30967e48030cFF

Currently running 13 validators with indexes 1256,401557,405852,405853,405855,405856,405857,405858,405859,405860,405865,405866,405867

**Comment by Infer114** [comment]
you are missing 3 others with 0xF3523fd7318C8Fb037eD081cDb478d45C70eD570

**Comment by ProSoloStaker** [comment]
> Hi, same here - I can’t figure out why validator #409907 is not in the list. Solo staker here with 2 validators, doing it directly without any third-party like Rocket Pool, Allnodes, etc. Thanks!

To add, withdrawal address is 0xe8637bB8ACf92391140bFb6570AAfCdbe20e70A1

Any chance this could be addressed for STKR airdrop please?

**Comment by ariskk** [comment]
Many (if not all) of them are now on the list, eg
https://github.com/rated-network/solo-stakers/blob/main/solo_stakers_v1.csv#L7550
https://github.com/rated-network/solo-stakers/blob/main/solo_stakers_v1.csv#L6523
https://github.com/rated-network/solo-stakers/blob/main/solo_stakers_v1.csv#L6113

**Comment by nelway42** [comment]
Yep confirmed mine is also included now.
